packages/material-ui-icons/scripts/download.js

```diff
@@ -1,42 +1,47 @@
 /* eslint-disable no-console */
 
+import 'isomorphic-fetch';
 import fse from 'fs-extra';
 import path from 'path';
 import yargs from 'yargs';
 import Queue from 'modules/waterfall/Queue';
 import sleep from 'modules/waterfall/sleep';
 import retry from 'modules/waterfall/retry';
-import 'isomorphic-fetch';
 
 const themeMap = {
   baseline: '', // filled
-  outline: '_outlined',
-  round: '_rounded',
-  twotone: '_two_tone',
-  sharp: '_sharp',
+  outline: 'outlined',
+  round: 'round',
+  twotone: 'twotone',
+  sharp: 'sharp',
+};
+
+const themeFileNameMap = {
+  baseline: '', // filled
+  outline: 'Outlined',
+  round: 'Rounded',
+  twotone: 'TwoTone',
+  sharp: 'Sharp',
```

Comment:
This makes for odd mixed-case file names:

```
ic_call_missed_outgoingTwoTone_24px.svg 
```

Appreciate they're only temp files, so it's probably just me.

Reply:
My approach probably should have been to name the string 'two_tone' and then just remove the '_' when fetching from google.

Reply:
Maybe we can keep a crazy convention internally, at least if we rarely need to come back to it 🙃.

Reply:
The rename filter takes care of snake case to camel case, so dropping the prefix and using "two_tone" etc should be no big deal.

But yes, it's internal, so we could keep it crappy if you prefer. :D

Reply:
mixed-case file names are gone!